spring-cloud-stream-binder-kinesis/src/test/java/org/springframework/cloud/stream/binder/kinesis/LocalKinesisResource.java

```diff
@@ -53,7 +49,6 @@ protected void obtainResource() throws Exception {
 		System.setProperty(SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY, "true");
 
 		this.resource = AmazonKinesisAsyncClientBuilder.standard()
-				.withCredentials(mock(AWSCredentialsProvider.class))
```

Comment:
Giving the client a stock proxy object doesn't provide any credentials despite the client expecting them. I was getting credential errors until I removed that line. 

I didn't dig deeper into the authentication mechanics of Kinesalite, but I confirmed the interactions were working without specifying credentials.